### PR TITLE
[IMP] account : remove analytic default modules

### PR DESCRIPTION
**Pull request by jbw-odoo**
Merge account_analytic_default into account
Merge account_analytic_default_hr_expense into hr
Merge account_analytic_default_purchase into purchase

Prevent having to manually install default modules after activating analytics

Task: 2182900

### CODE REVIEW

File: addons/purchase/models/purchase.py [hunk 1]
```diff
@@ -657,7 +657,7 @@ class PurchaseOrderLine(models.Model):
 
     partner_id = fields.Many2one('res.partner', related='order_id.partner_id', string='Partner', readonly=True, store=True)
     currency_id = fields.Many2one(related='order_id.currency_id', store=True, string='Currency', readonly=True)
-    date_order = fields.Datetime(related='order_id.date_order', string='Order Date', readonly=True)
+    date_order = fields.Datetime(related='order_id.date_order', string='Order Date', readonly=True, store=True)
```
Comment (qdp-odoo) [inline addons/purchase/models/purchase.py:660]: where does that diff come from?

Comment (jbw-odoo) [reply addons/purchase/models/purchase.py:660]: I had a db error during my dev when it didn't find the field in DB. Did not realise that after code modifications it was not need anymore.

**Comment by jbw-odoo** [comment]
@qdp-odoo this & migration updated

**Comment by qdp-odoo** [comment]
just need to solve the conflicts now ;-)

**Comment by jbw-odoo** [comment]
> just need to solve the conflicts now ;-)
@qdp-odoo resolved :)

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by gonzalolenzi** [comment]
@jbw-odoo @qdp-odoo Hi, when removing the analytic default module, the possibility to set default analytic accounts by User got lost.  Do you plan to restore it? Thanks in advance.

**Pull request closed without merging**